### PR TITLE
Port reports ui changes to glitch-soc flavour

### PR DESCRIPTION
**Pull request by ThibG**

### CODE REVIEW

File: app/javascript/flavours/glitch/features/report/components/status_check_box.js [hunk 1]
```diff
@@ -14,18 +18,48 @@ export default class StatusCheckBox extends React.PureComponent {
...
+
     return (
       <div className='status-check-box'>
-        <div
-          className='status__content'
-          dangerouslySetInnerHTML={content}
-        />
+        <div className='status-check-box__status'>
+          <StatusContent status={status} />
+          {media}
```
Comment (marrus-sh) [inline app/javascript/flavours/glitch/features/report/components/status_check_box.js]: With `glitch` we usually put media *inside* of `<StatusContent>`; this should probably be

```jsx
<StatusContent
  media={media}
  status={status}
/>
```

It's not a necessity, but setting and passing through a `mediaIcon` like in https://github.com/glitch-soc/mastodon/blob/master/app/javascript/flavours/glitch/components/status.js#L317-L344 would make the CW appearance more consistent as well.

Comment (ThibG) [reply app/javascript/flavours/glitch/features/report/components/status_check_box.js]: Ok. Done!

**Comment by ThibG** [comment]
Ping?

**Comment by yipdw** [comment]
I'm deploying this to a production instance; if the ported code works fine, I think we can merge this.

**Comment by yipdw** [comment]
The send-a-copy-of-this-report-to-the-admin UI change works for me; merging.

**Pull request merged**